### ISSUE TITLE
Won't work with new ISP

### ISSUE THREAD

**Issue by GitTheHellOutaHere** [jedisct1/dnscrypt-proxy#437]
I've been staying in a hotel for a month and connecting to a secured signal from their router.
I accessed the Internet through their capture portal and have been using DNSCrypt all along.

Tonight the hotel changed their ISP to [WiFi Now Networks](http://www.wifinownet.net) and now have an open signal. 
DNSCrypt no longer connects to any server. I can still use Viscosity to connect to OpenVPN servers though
The new default DNS servers are 75.75.75.75, 75.75.75.76, 2001:558:feed::1, 2001:558:feed::2.
It doesn't work with the free DNS servers I was using before.

Any suggestions?

Please don't tell me to script something or edit config files unless you can provide step-by-step instructions - I'm not a networking expert (which should be obvious).

**Comment by jedisct1** [comment]
Did you try the TCP mode (`--tcp-only`)?

**Comment by GitTheHellOutaHere** [comment]
Don't know what that is.
DNSCrypt has very limited options in its preferences and that's not one of them.

So I Googled it and found this:
_If you have a firewall or other middleware mangling your packets, you should try enabling DNSCrypt with TCP over port 443.  This will make most firewalls think it’s HTTPS traffic and leave it alone._

But no instructions how to accomplish this.

**Comment by jedisct1** [comment]
Got it, you are using the OSXClient app for Mac.

That app automatically tries to use TCP, so if it can't establish an authenticated connection, chances are that you won't be able to use DNSCrypt from this network at all.

You may want to try [DNSSEC-Trigger](https://www.nlnetlabs.nl/projects/dnssec-trigger/) instead, that does exactly the same thing, but using a different protocol (similar to HTTPS).

**Comment by GitTheHellOutaHere** [comment]
Thanks for the suggestion, but I don't even understand what they're saying on the homepage. 😕

**Comment by jedisct1** [comment]
[Click here](https://www.nlnetlabs.nl/downloads/dnssec-trigger/dnssectrigger-0.12.dmg) to download the version for Mac.

**Comment by GitTheHellOutaHere** [comment]
Downloading it isn't the problem - the link is right on top of the page.

Understanding what it _does_ is the problem. Is it a replacement for DNSCrypt or something I'd run only when DNSCrypt can't connect via UDP?

I don't understand a lot of what they're talking about on that page - don't get Unbound or any of this jargon: _Check open resolver over SSL port 443(https port). If that works, unbound is told to use SSL encapsulated DNS over port 443 to an open (DNSSEC capable) resolver._

I am connecting to the alternate-dns.com servers I set in System Preferences, so this hotel's ISP is allowing me to use other DNS servers. Don't know why DNSCrypt can't connect.
Is there a test I should run to see if some port is blocked?

**Comment by jedisct1** [comment]
It's a replacement for the application you are trying to use, that might work on your network.

**Comment by GitTheHellOutaHere** [comment]
Well, that was completely futile.

After running a backup I installed the app.
There is no front-end with it, so I have a bunch of files installed that I can't use.

**Welcome to the DnssecTrigger for MacOS X installer.**

**This installer installs: libldns, unbound and dnssec-trigger.  It starts a validating DNSSEC cache on this system.**

**The configuration files are /etc/unbound/unbound.conf and /etc/dnssec-trigger/dnssec-trigger.conf**

**The trust anchor for the Internet root zone will be downloaded and configured with unbound-anchor if the file /etc/unbound/root.key does not exist.**

**Information about DnssecTrigger can be found at**
**http://www.nlnetlabs.nl/projects/dnssec-trigger**
**Information about Unbound can be found at**
**http://unbound.net**

**Comment by GitTheHellOutaHere** [comment]
Don't know why, but DNSCrypt connected after reboot today. 😎

I did change the DNS servers in System Preferences earlier, but it still wasn't connected.
I rebooted 30 minutes ago and saw the padlock pop up.

I reboot daily, often more than once, so it's had plenty of chances before now to connect.

That's great because I've wasted a LOT of time researching and experimenting with other software.
There doesn't seem to be anything else that's as easy to use as DNSCrypt-OSXClient.
